CMakeLists.txt

```diff
@@ -2,6 +2,11 @@ cmake_minimum_required(VERSION 2.8.3)
 project(hip)
 
 #############################
+# Options
+#############################
+option(BUILD_HIPIFY_CLANG "Enable building the CUDA->HIP converter" ON)
```

Comment:
Make BUILD_HIPIFY_CLANG OFF by default as it was before, moreover this change breaks HIP build on Jenkins due to LLVM not found.

```
[hcc_ctu] CMake Error at hipify-clang/CMakeLists.txt:4 (find_package):
[hcc_ctu]   By not providing "FindLLVM.cmake" in CMAKE_MODULE_PATH this project has
[hcc_ctu]   asked CMake to find a package configuration file provided by "LLVM", but
[hcc_ctu]   CMake did not find one.
[hcc_ctu] 
[hcc_ctu]   Could not find a package configuration file provided by "LLVM" with any of
[hcc_ctu]   the following names:
[hcc_ctu] 
[hcc_ctu]     LLVMConfig.cmake
[hcc_ctu]     llvm-config.cmake
[hcc_ctu] 
[hcc_ctu]   Add the installation prefix of "LLVM" to CMAKE_PREFIX_PATH or set
[hcc_ctu]   "LLVM_DIR" to a directory containing one of the above files.  If "LLVM"
[hcc_ctu]   provides a separate development package or SDK, be sure it has been
[hcc_ctu]   installed.
[hcc_ctu] 
[hcc_ctu] 
[hcc_ctu] -- Configuring incomplete, errors occurred!
```